flexget/components/notify/notifiers/matrix.py

```diff
@@ -0,0 +1,78 @@
+from loguru import logger
+from requests.exceptions import RequestException
+
+from flexget import plugin
+from flexget.event import event
+from flexget.plugin import PluginWarning
+from flexget.utils.requests import Session as RequestSession
+
+requests = RequestSession(max_retries=3)
+
+plugin_name = 'matrix'
+
+logger = logger.bind(name=plugin_name)
+
+
+def urljoin(*args):
```

Comment:
why not just use standard urljoin ?

Reply:
urllib.urljoin() takes from 2 to 3 positional arguments but 4 were required.